modules/lfs/locks.go

```diff
@@ -44,7 +45,7 @@ func checkIsValidRequest(ctx *context.Context, post bool) bool {
 	return true
 }
 
-func handleLockListOut(ctx *context.Context, lock *models.LFSLock, err error) {
+func handleLockListOut(ctx *context.Context, repo *models.Repository, lock *models.LFSLock, err error) {
```

Comment:
Why we need a second repo but not use ctx.Repo.Repository?

Reply:
ctx.Repo.Repository is not set.

Reply:
@lunny - ctx.Repo and ctx.Repo.Repository are only set if you use context.RepoAssignment which is inappropriate for LFS, as LFS has its own way of determining authentication.